packages/config-cli/src/cli/screenshot/screenshot.ts

```diff
@@ -39,104 +45,88 @@ export class CommandScreenShot extends CommandLineAction {
   }
 
   protected onDefineParameters(): void {
+    this.config = this.defineStringParameter({
+      argumentName: 'CONFIG',
+      parameterLongName: '--config',
+      description: 'Path of config bundle file, could be both local file or s3.',
+    });
+
     this.host = this.defineStringParameter({
       argumentName: 'HOST',
       parameterLongName: '--host',
       description: 'Host to use',
-      defaultValue: 'basemaps.linz.govt.nz',
-    });
-
-    this.tag = this.defineStringParameter({
-      argumentName: 'TAG',
-      parameterShortName: '-t',
-      parameterLongName: '--tag',
-      description: 'PR tag(PR-number) or "production"',
-      defaultValue: 'production',
+      defaultValue: DefaultHost,
     });
 
     this.tiles = this.defineStringParameter({
       argumentName: 'TILES',
       parameterLongName: '--tiles',
       description: 'JSON file path for the test tiles',
-      defaultValue: './test-tiles/default.test.tiles.json',
+      defaultValue: DefaultTestTiles,
     });
   }
 
   async onExecute(): Promise<void> {
     const logger = LogConfig.get();
+    const config = this.config.value;
+    let host = this.host.value ?? DefaultHost;
+    const tiles = this.tiles.value ?? DefaultTestTiles;
+
+    let BasemapsServer: FastifyInstance | undefined = undefined;
+    if (config != null) {
+      const port = await getPort({ port: portNumbers(10000, 11000) });
+      host = `http://localhost:${port}`;
+      BasemapsServer = createServer(logger);
+
+      if (BasemapsServer == null) throw new Error('Failed to Create server with the config File');
+      BasemapsServer.listen(port, '0.0.0.0', () => {
```

Comment:
this is not waiting for this to finish loading.

Reply:
Sorry, I forget this. Just fixed.